src/reactviews/pages/QueryResult/queryResultPane.tsx

```diff
@@ -171,7 +170,92 @@ export const QueryResultPane = () => {
         return null;
     }
 
-    const gridRef = useRef<SlickGridHandle>(null);
+    const gridRef = useRef<ResultGridHandle>(null);
+
+    const renderGrid = (idx: number) => {
```

Comment:
This is to extract grid rendering logic into a helper function, not much code change (except for styling + multi result set suppot)